From: Denis Kirjanov <dkirjanov@kernel.org>
Date: Sat, 30 Oct 2010 16:24:25 -0700
Subject: [PATCH 41/56] netxen_nic: Fix the tx queue manipulation bug in
 netxen_nic_probe
Git-commit: c117e4a2bb49116b0b51df06bbabe638d28dfae7
Patch-mainline: v2.6.37-rc1

We should not stop the egress queue during probe because it is wrong.

Signed-off-by: Denis Kirjanov <dkirjanov@kernel.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Jeff Mahoney <jeffm@suse.com>
---
 drivers/net/netxen/netxen_nic_main.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/net/netxen/netxen_nic_main.c b/drivers/net/netxen/netxen_nic_main.c
index b6c8d29..1be7729 100644
--- a/drivers/net/netxen/netxen_nic_main.c
+++ b/drivers/net/netxen/netxen_nic_main.c
@@ -1239,7 +1239,6 @@ netxen_setup_netdev(struct netxen_adapter *adapter,
 		dev_warn(&pdev->dev, "failed to read mac addr\n");
 
 	netif_carrier_off(netdev);
-	netif_stop_queue(netdev);
 
 	err = register_netdev(netdev);
 	if (err) {
-- 
1.6.3.3

